### DISCUSSION TITLE
Trying to maximize cheap electricity price

### DISCUSSION THREAD

**Discussion by mjo75** [gekkekoe/esphome-ecodan-hp#83] [Announcements]
First of all thank you for making this great project. Actually inspired me to get Ecodan instead of Daikin. 

For me electricity price changes hourly (cold be 3cents during night and 20-60c during the day. Can't beat that with any COP. 

With an old boiler I cranked my DHW to max (85C) during the cheapest hours and the water lasted till the next night for the whole family.

Now the only way is to put legionella (70C) program on every night, but the time has to be set manually from the inside unit. 

Is there a way to control that time with ESP?

Mikko

**Comment by merlin051** [comment]
Couldn't you just set it to higher than the max flow Temp and the immersion will kick in for support?

**Reply by mjo75** [reply]
Could you advice how to set it higher than 60 C. I did not find any other way than using legionella program

**Comment by gekkekoe** [comment]
How much liters is your tank ? What is the tank temp at the end of the day? I mean 45c is also fine for a shower?

I think I limited it at 60c in the thermostat https://github.com/gekkekoe/esphome-ecodan-hp/blob/95c3d4cc1c40fcac6ece915343dd543bad97968e/components/ecodan/climate.cpp#L171, you could adjust it locally, but i'm unsure if the cn105 allows it.

The legionella programme cannot be controlled. But you should be able set up a schedule to have it run once a day at 70c?

**Reply by mjo75** [reply]
Thanks for quick reply.

**Reply by mjo75** [reply]
Good to know that legionella program cannot be controlled. Actually what is maybe good to know... I changed the scheduled time from 3am to 1am. And as the period is once a day it did not turn on last night as the time between was less than 24h :).

I have to find another way to maximally take benefit of the cheapest hours.

**Reply by gekkekoe** [reply]
for regular dhw, wat is the max temp that you can set on the ftc controller? I think it's 60c?

**Reply by mjo75** [reply]
Yes. 60 is max. The tank is 200L. The family is quite big. Two adults, 2y 6y, 10y and 16y kids and 19y visits home quite often. The last two are pretty unsatisfied with cold shower 😃 .
